tests/framework/framework.go

```diff
@@ -64,10 +64,15 @@ type Framework struct {
 
 // initialize run-time flags
 func init() {
-	flag.StringVar(&kubectlPath, "kubectl-path", "", "Set path to kubectl binary")
-	flag.StringVar(&cdiInstallNs, "installed-namespace", "kube-system", "Set the namespace CDI is installed in")
```

Comment:
So as far as I understand these 4 lines, its to make sure these arguments are expected and to set a default value if not found like the installed-namespace (the name space the cdi controller is installed in). So if you don't pass in values you get an error, which looks like the error you were getting.